openpype/hosts/fusion/api/lib.py

```diff
@@ -302,10 +302,25 @@ def get_frame_path(path):
     return filename, padding, ext
 
 
-def get_current_comp():
-    """Hack to get current comp in this session"""
+def get_fusion():
```

Comment:
This looks like a function from Nuclear power plant :)
```suggestion
def get_fusion_module():
```